tests/foreman/api/test_contentview.py

```diff
@@ -1722,3 +1722,33 @@ def test_positive_publish_promote_with_RH_ostree_and_other(
         content_view.publish()
         promote(content_view.read().version[0], module_lce.id)
         assert len(content_view.read().version[0].read().environment) == 2
+
+    @pytest.mark.stubbed
+    @pytest.mark.tier1
+    def test_positive_check_pulp3_is_installed(self):
+        """Ensure that pulp3 is installed
+
+        :id: 5d5690bf-56c7-4dba-bb09-de0f5f4861b1
+
+        :expectedresults: Pulp3 is active
+
+        :CaseLevel: System
+
+        :CaseImportance: Critical
+        """
+        pass
+
+    @pytest.mark.stubbed
+    @pytest.mark.tier1
+    def test_positive_check_pulp2_is_removed(self):
```

Comment:
Do we have any API for pulp2 removal package verification?

Reply:
I haven't checked, but if the API hits pulp3, then we know pulp2 is not present.  There's no other real good way to verify this other than try and connect to pulp2 db.

Reply:
yes, we need to check it, if the API is there then we are good with this just need to update the test steps otherwise we can cover it in the CLI part.